src/sinks/datadog/metrics/encoder.rs

```diff
@@ -182,10 +189,20 @@ impl DatadogMetricsEncoder {
             compressed_limit,
             state: EncoderState::default(),
             log_schema: log_schema(),
+            origin_product_value: determine_origin_product_value(),
         })
     }
 }
 
+fn determine_origin_product_value() -> u32 {
+    option_env!("ORIGIN_PRODUCT")
```

Comment:
Is this env var something we have named? I'm wondering if we should namespace it `DD_ORIGIN_PRODUCT` or `VECTOR_ORIGIN_PRODUCT`?

Reply:
👍 to `DD_ORIGIN_PRODUCT`.

Reply:
Just noting that the Agent uses `DD_` as the prefix for it's env variables which are particularly relevant when using the Agent in a container. Perhaps we should make this `OPW_` to not cause confusion?